src/site/content/en/blog/user-agent-client-hints/index.md

```diff
@@ -0,0 +1,434 @@
+browsers for historical reasons.
+
+The combination of these parameters with the sheer diversity of possible values
+means the User-Agent string could contain enough information to allow individual
+users to be uniquely identified. If you test your own browser at
+[AmIUnique](https://amiunique.org/), you can see just how closely **your**
+User-Agent string identifies **you**. The lower your resulting "Similarity
+ratio" is, the more unique your requests are, the easier it is for servers to
+covertly track you.
+
+The User-Agent string enables many legitimate [use
+cases](https://github.com/WICG/ua-client-hints/blob/master/README.md#use-cases),
+and serves an important purpose for developers and site owners. However, it is
+also critical that users' privacy is protected against any covert tracking
+methods, and sending UA information by default goes against that goal.
+
+There's also a need to improve web compatibility when it comes to the User-Agent
+string. It is unstructured, so parsing it results in unnecessary complexity,
+which is often the cause for bugs and site compatibility issues that hurt users.
+These issues also disproportionately hurt users of less common browsers, as
+sites may have failed to test against their configuration.
+
+## Introducing the new User-Agent Client Hints
+
+[User-Agent Client
+Hints](https://github.com/WICG/ua-client-hints#explainer-reducing-user-agent-granularity)
+enable access to the same information but in a more privacy-preserving way, in
+turn enabling browsers to eventually reduce the User-Agent string's default of
+broadcasting everything. [Client
+Hints](https://tools.ietf.org/html/draft-ietf-httpbis-client-hints) enforce a
+model where the server must ask the browser for a set of data about the client
+(the hints) and the browser applies its own policies or user configuration to
+determine what data is returned. This means that rather than exposing **all**
+the User-Agent information by default, access is now managed in an explicit and
+auditable fashion. Developers also benefit from a simpler API - no more regular
+expressions!
+
+The current set of Client Hints primarily describes the browser's display and
+connection capabilities. You can explore the details in [Automating Resource
+Selection with Client
+Hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints),
+but here's a quick refresher on the process.
+
+The server asks for specific Client Hints via a header:
+
+⬇️ _Response from server_
+
+```text
+Accept-CH: Viewport-Width, Width
+```
+
+Or a meta tag:
+
+```html
+<meta http-equiv="Accept-CH" content="Viewport-Width, Width" />
+```
+
+The browser can then choose to send the following headers back in subsequent
+requests:
+
+⬆️ _Subsequent request_
+
+```text
+Viewport-Width: 460
+Width: 230
+```
+
+The server can choose to vary its responses, for example by serving images at an
+appropriate resolution.
+
+{% Aside 'objective' %}
```

Comment:
```suggestion
{% Aside %}
```
Using the "objective" styling is a bit of a stretch here and distracted me. We usually use that to indicate the main objective of a tutorial. Removing `'objective'` will style this as a [note aside](https://web.dev/handbook/web-dev-components/#note-asides)

![image](https://user-images.githubusercontent.com/4713486/85611677-06045a00-b60d-11ea-839d-fca789035484.png)